administrator/components/com_installer/views/update/tmpl/default.php

```diff
@@ -105,6 +105,15 @@
 									</td>
 									<td>
 										<span class="badge badge-success"><?php echo $item->version; ?></span>
+										<?php
+										$modal_params = array();
```

Comment:
if you are in a layout file can you please use opening and closing PHP comands for each line?
So this line becomes
`<?php $modal_params = array(); ?>`